### PR TITLE
Chore/update-abi-staking-2

### LINKED ISSUE #222: Update abi with latest field

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Resolve #222

### What change is being made?

Update the ERC20CollateralPool ABI to include new events and functions related to user position liquidation and maximum lending limits, and modify the error messages and tests to reflect these changes.

### Why are these changes being made?

These changes are being made to enhance the functionality of the ERC20CollateralPool by introducing a mechanism to handle user position liquidation and enforce a maximum lending limit. This ensures better management of collateral pools and prevents over-lending, which could lead to financial instability within the system. The updates to the error messages and tests ensure that these new features are properly integrated and validated.

> Is this description stale? Ask me to generate a new description by commenting `/korbit-generate-pr-description`

### CODE REVIEW

File: src/errors/error-messages.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const erc20CollateralPoolErrorMessage = {
   maxLimitAllowed: 'Max limit allowed is 1000',
   endTimeReached: 'Pool has ended',
   poolIsNotClosed: 'Pool is not closed',
+  maxLendedIsReached: 'Max lended is reached',
```
Comment (xavier506) [inline src/errors/error-messages.ts:27]: Past tense of Lend is Lent 

Max Lent

Comment (xavier506) [reply src/errors/error-messages.ts:27]: maxLendedIsReached: 'Maximum lending limit reached'

Comment (leisterfrancisco) [reply src/errors/error-messages.ts:27]: Sure, you're right, I followed same contract names but I'll correct it, thanks.

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]

**Pull request merged**